spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -200,44 +206,44 @@ public class DefaultTaskExecutionService implements TaskExecutionService {
 	 */
 	@Deprecated
 	public DefaultTaskExecutionService(
-		PropertyResolver propertyResolver,
-		LauncherRepository launcherRepository,
-		AuditRecordService auditRecordService,
-		TaskRepositoryContainer taskRepositoryContainer,
-		TaskExecutionInfoService taskExecutionInfoService,
-		TaskDeploymentRepository taskDeploymentRepository,
-		TaskDefinitionRepository taskDefinitionRepository,
-		TaskDefinitionReader taskDefinitionReader,
-		TaskExecutionCreationService taskExecutionRepositoryService,
-		TaskAppDeploymentRequestCreator taskAppDeploymentRequestCreator,
-		AggregateTaskExplorer taskExplorer,
-		DataflowTaskExecutionDaoContainer dataflowTaskExecutionDaoContainer,
-		DataflowTaskExecutionMetadataDaoContainer dataflowTaskExecutionMetadataDaoContainer,
-		DataflowTaskExecutionQueryDao dataflowTaskExecutionQueryDao,
-		OAuth2TokenUtilsService oauth2TokenUtilsService,
-		TaskSaveService taskSaveService,
-		TaskConfigurationProperties taskConfigurationProperties,
-		AggregateExecutionSupport aggregateExecutionSupport
+			PropertyResolver propertyResolver,
```

Comment:
Please remove all the extra formatting. I am sure IntelliJ did it and you did not notice it.

Reply:
Yes I pressed the shortcut for reformat for the whole class which leads to this noise.